mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/modules/CassandraMessageModule.java

```diff
@@ -114,6 +116,7 @@ public interface CassandraMessageModule {
         .statement(statement -> types -> statement
             .withPartitionKey(CassandraMessageIds.MESSAGE_ID, TIMEUUID)
             .withColumn(CassandraMessageV3Table.INTERNAL_DATE, TIMESTAMP)
+            .withColumn(CassandraMessageV3Table.SAVE_DATE, TIMESTAMP)
```

Comment:
Save date is mailbox specific. There is no reasons to store it in a cassandra table that is not mailboxe scopped (we should not store it in messagev3)